### PR TITLE
Updating story examples with new step spelling

### PR DESCRIPTION
**Pull request by VitaliLebedzeu**

### CODE REVIEW

**Review by uarlouski — APPROVED** [review]

File: src/main/resources/story/web_app/Google Search.story [hunk 1]
```diff
@@ -7,8 +7,8 @@ Examples:
 |search-button      |By.xpath((//*[@name='btnK'])[1])|
 
 Scenario: Verify VIVIDUS documentation is in Top 10 Google search results
-Given I am on a page with the URL 'https://www.google.com/'
+Given I am on page with URL `https://www.google.com/`
 When I enter `VIVIDUS test automation` in field located `<search-input-field>`
-When I wait until element located `<search-button>` appears
+When I wait until element located by `<search-button>` appears
 When I click on element located `<search-button>`
```
Comment (valfirst) [inline src/main/resources/story/web_app/Google Search.story:13]: would you mind updating all steps to their non-deprecated alternatives?

Comment (VitaliLebedzeu) [reply src/main/resources/story/web_app/Google Search.story]: All deprecated steps are updated

Comment (valfirst) [reply src/main/resources/story/web_app/Google Search.story]: https://github.com/vividus-framework/vividus/blob/master/vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/ElementSteps.java#L294-L295

Comment (VitaliLebedzeu) [reply src/main/resources/story/web_app/Google Search.story]: It's correct and as I see in my changes
image

File: src/main/resources/story/rest_api/Star Wars API.story [hunk 2]
```diff
@@ -2,5 +2,5 @@ Description: Test demoing VIVIDUS capabilities for REST API
 
 Scenario: Verify Luke's eyes are blue
 When I execute HTTP GET request for resource with URL `https://swapi.dev/api/people/1/`
-Then `${responseCode}` is equal to `200`
-Then JSON element value from context by JSON path `$.eye_color` is equal to `blue`
+Then `${response-code}` is equal to `200`
```
Comment (valfirst) [inline src/main/resources/story/rest_api/Star Wars API.story:5]:
```suggestion
Then `${responseCode}` is equal to `200`
```
https://docs.vividus.dev/vividus/latest/commons/variables.html:
> Dynamic variables support both lower camel case (e.g. ${dynamicVariableName}) and lower hyphen case (e.g. ${dynamic-variable-name}) name formats.

**Pull request merged**